server/index.js

```diff
@@ -16,6 +17,14 @@ app.use(express.static('client/public'));
 app.use(express.json());
 app.use(middleware.logger);
 
+app.get('/google/auth', (req, res) => {
+  if (process.env.GOOGLE_API) {
+    res.send(process.env.GOOGLE_API)
```

Comment:
We could the actual call to the Google Maps API here too if we want (e.g., passing the request through). But this works too.

Reply:
I ad considered doing that, and while it would work for the geocode request, the package that manages the react portion of google maps uses a special loadscript component that takes the key as a prop; figured it would just be easier to do the call once in the map file then use the variable accordingly.